### DISCUSSION TITLE
I'm sorry to request that I want to refund

### DISCUSSION THREAD

**Discussion by linyang2124** [aclap-dev/video-downloadhelper#1322] [Help and Q&A]
### Your question

For that it cannot download videos from NHK 

### Addon & CoApp version and details

version: 9.1.0.44
target: google
channel: stable
lang: zh-CN
coapp: {"found":true,"new_version":false,"path":"D:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"2d7f31e08d8d41f3"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 3
}

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
